_publications/jors/paper.tex

```diff
@@ -0,0 +1,362 @@
+it can be deployed on GitHub's Pages infrastructure, which enables
+streamlined deployment without the need for any back-end
+infrastructure.
+
+\section*{Programming language}
+
+PFHub is currently built and tested using the programming languages
+and versions outlined in Table~\ref{tab:versions}.
+
+\begin{table}[h!]
+  \centering
+  \caption{PFHub programming languages and corresponding supported
+    versions.}
+  \begin{tabular}{|l|l|}
+    \hline
+    Language         & Version \\
+    \hline
+    HTML             & 5       \\
+    Jupyter Notebook & 5.4.0   \\
+    JavaScript       & 5       \\
+    Nix              & 2.1.3   \\
+    CoffeeScript     & 1.12.7  \\
+    CSS              & 4       \\
+    \hline
+  \end{tabular}
+  \label{tab:versions}
+\end{table}
+
+
+\section*{Additional system requirements}
+
+There are no additional system requirements.
+
+\section*{Dependencies}
+
+The entire environment can be built using the Nix Package Manager so
+the only required dependency is a functional Nix installation. The
+PFHub framework has over 2000 separate package dependencies using
+data from the Nix package manager. The full dependency graph for PFHub
+can be seen online~\cite{dependencies}.
+
+\section*{List of contributors}
+
+1. Wheeler, Daniel; A, \githublink{wd15} \\
+2. Keller, Trevor; A, \githublink{tkphd} \\
+3. DeWitt, Stephen J.; E, \githublink{stvdwtt} \\
+4. Jokisaari, Andrea M.; D, \githublink{amjokisaari} \\
+5. Schwen, Daniel; D, \githublink{dschwen} \\
+6. Guyer, Jonathan E.; A, \githublink{guyer} \\
+
+See the contributors list on GitHub~\cite{contributors}.
+
+\section*{Software location:}
+
+{\bf Archive}
+
+\begin{description}[noitemsep,topsep=0pt]
+	\item[Name:] Zenodo
+	\item[Persistent identifier:]
+          \href{https://dx.doi.org/10.5281/zenodo.2592705}{10.5281/zenodo.2592705}
+	\item[Licence:] NIST Software License~\cite{nistlicense}
+	\item[Publisher:]  Daniel Wheeler
+	\item[Version published:] v0.1
+	\item[Date published:] 13/03/19
+\end{description}
+
+{\bf Code repository}
+
+\begin{description}[noitemsep,topsep=0pt]
+	\item[Name:] GitHub
+	\item[Persistent identifier:] \url{https://github.com/usnistgov/pfhub/tree/v0.1}
+	\item[Licence:] NIST Software License~\cite{nistlicense}
+	\item[Date published:] 13/03/19
+\end{description}
+
+\section*{Languages}
+
+English
+
+\section*{(3) Reuse potential}
+
+The website infrastructure can be cloned as a Git repository or
+downloaded as a ZIP archive and deployed with minimum effort. The
+mechanism for uploading data using Staticman can be easily configured
+for a new repository location. However, customizing the content of the
+website for a particular scientific community would require
+considerable effort. The following steps are the more challenging
+aspects of deploying the framework for a new community.
+
+\begin{itemize}
+  \item Upload new benchmark definitions in a format that Jekyll can
```

Comment:
Is it really limited to benchmark problems?  If that's the case, that needs to be made clear in the abstract and introduction.

Reply:
Changed to "new data upload specifications"